apps/E2E/src/ButtonV1/pages/ButtonV1PageObject.ts

```diff
@@ -4,9 +4,9 @@ import {
   BUTTON_NO_A11Y_LABEL_COMPONENT,
   HOMEPAGE_BUTTON_BUTTON,
   BUTTON_ON_PRESS,
-} from '../../Button/consts';
+} from '../../ButtonLegacy/consts';
```

Comment:
Should we create a consts file for ButtonV1?

Reply:
Same for IconV1

Reply:
I would say that's out of the scope of this PR. For the Button, there's only one const page because there was only one test folder for the button where the consts used to live, so both legacy and V1 button consts were in the same file.